doc/sphinx/source/recipes/recipe_consecdrydays.rst

```diff
@@ -40,6 +40,13 @@ User settings in recipe
 
    * frlim: the shortest number of consecutive dry days for entering statistic on frequency of dry periods.
 
+   *Optional settings (script)*
+
+   Under ``plot``:
+
+   * cmap: the name of a colormap. cmocean colormaps are also supported.
+
+   * other matplotlib plot keyword arguments can also be supplied.
```

Comment:
I would mention here that these are keyword arguments for
```
:func:`esmvaltool.diag_scripts.shared.plot.global_pcolormesh`
```

Reply:
Thanks for the review! Done